src/components/RowSelection/index.js

```diff
@@ -27,6 +64,22 @@ export default class RowSelection extends PureComponent {
           disabled={!(selectedItems > 0)}
           style={{ marginRight: 8 }}
         />
+        <Dropdown
+          onSelect={this.onSelectManageRunDropdown}
+          toggle={
+            <DropdownToggle
+              onToggle={this.onToggleManageRunDropdown}
+              toggleIndicator={CaretDownIcon}
+              isPrimary
+              id="toggle-id-4"
```

Comment:
We can remove the id prop for this component if not required by PatternFly.

Reply:
removed!